### DISCUSSION TITLE
Capture Event when start to draw

### DISCUSSION THREAD

**Discussion by GonzaCS** [warting/android-signaturepad#103] [Q&A]
Is there any event that I can capture when someone starts drawing?

**Comment by warting** [comment]
Not yet. But let's create one if needed! Please create an issue and I can make it happen

**Comment by warting** [comment]
Now there is :)

**Reply by eniz1806** [reply]
hello. Why android studio is not suggesting se.warting....for UI lib is there imported but this happening..

https://ibb.co/k8YV9b5
